src/EA.Weee.Web/Areas/AATFReturn/Controllers/NonObligatedController.cs

```diff
@@ -63,5 +74,22 @@ private async Task SetBreadcrumb(Guid organisationId, string activity)
             breadcrumb.ExternalActivity = activity;
             breadcrumb.SchemeInfo = await cache.FetchSchemePublicInfo(organisationId);
         }
+
+        private async Task<bool> ValidateResult(NonObligatedValuesViewModel model, IWeeeClient client)
```

Comment:
This can return void as we don't need to use the boolean return as we just check the model state